src/navigation/optOutFooter.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+// this provides an opt-out link in the footer for the compulsory opt-in phase of Next
+module.exports = function (items, currentUrl) {
+	const optOutLink = {
+		name: 'Old FT.com',
```

Comment:
I'd favour adding a `trackableName` (or similar) for use in the `data-trackable` attributes, rather than reusing `name`. That way if the copy changes, we don't have to change our beacon queries to include the amended names.